### ISSUE TITLE
[ALM Accelerator - Feature]: Cross Tenant AA4PP functionality

### ISSUE THREAD

**Issue by megel** [microsoft/coe-starter-kit#3946]
### Is your feature request related to a problem? Please describe.

Currently it is not possible to use AA4PP in a X-Tenant setup. Each component must be in the same Azure Tenant. 

In detail:
* all AA4PP Dataverse environments **must** be in the same Azure Tenant
* AzDevOps **must** be in the same Azure Tenant

This avoid a couple of common scenarios:
1. MSFT Partner use & manage AzDevOps for AA4PP in Partner Azure Tenant to develop the customer, Customer owns the Power Platform environments in his Azure Tenant. As result, the AA4PP Canvas App can't be used to start an export / import / ....
1. Huge Companies with multiple Azure Tenants (AzDevOps, Power Platform Europe, Asia, ...) can't use AA4PP across their tenants. Same issue, because of AA4PP can't communicate with AzDevOps or target PP environments.

### Describe the solution you'd like

I guess it would be possible to use AA4PP satellite solutions which act as a Proxy between AA4PP environment & AzDevOps or target PP environment. Here is a blueprint:

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/15800069/195581199-d2b976f6-7728-4910-afd3-e5515f36f0d5.png">

### Describe alternatives you've considered

Manual Work 🤯 or self scription Pipelines with manual work.

@jenschristianschroder, @mikefactorial 

AB#1179

**Comment by cstriok** [comment]
I would also like to have this feature. It would save us a lot of manual setup work. thx

**Comment by megel** [comment]
Found this: #3161 it's from @SabrinaDiBartolomeo.

**Comment by ollimenzel** [comment]
I second this, would be really helpful

**Comment by sdhsynsci** [comment]
We need this also.

**Comment by stefanjackmuth** [comment]
Yes, we also need that feature. Currently we've set up a lot of manual pipelines - always a bit buggy as there is a lot of manual work to implement the correct settings like environment variables, connections etc.

**Comment by SBichler** [comment]
I had to set up AA4PP where DevOps was in our tenant and the Dataverse environments in the customer's tenant (which is very common at our company).
I had to do all steps manually and it was really complicated and time-consuming.
Would be awesome if this scenario would be considered in the setup process.

**Comment by pSturm799** [comment]
Sounds nice. Would be very helpful

**Comment by sdhsynsci** [comment]
> I had to set up AA4PP where DevOps was in our tenant and the Dataverse environments in the customer's tenant (which is very common at our company). I had to do all steps manually and it was really complicated and time-consuming. Would be awesome if this scenario would be considered in the setup process.

@SBichler In practice does this work for you?  I do not mind the manual setup (we had to do it for our internal projects because we already had projects and pipelines so the automatic stuff failed).

**Comment by SBichler** [comment]
> > I had to set up AA4PP where DevOps was in our tenant and the Dataverse environments in the customer's tenant (which is very common at our company). I had to do all steps manually and it was really complicated and time-consuming. Would be awesome if this scenario would be considered in the setup process.
> 
> @SBichler In practice does this work for you? Did you still install the AA4PP app in the client tenant? I do not mind the manual setup (we had to do it for our internal projects because we already had projects and pipelines so the automatic stuff failed).

@sdhsynsci yes it is currently running with one of our clients in a multi-tenant setup. I followed the manual [setup guide](https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-almacceleratorpowerplatform) of the docs for the most parts. It got a bit tricky with the AAD app registration. You must create one for each tenant and split up the API permissions accordingly. Also the user of our tenant was added as guest user to the customers tenant.

**Comment by AmarSaiRam** [comment]
According to https://learn.microsoft.com/en-us/power-platform/alm/pipelines, Power Platform Pipelines are now the recommended approach for Application Lifecycle Management (ALM). The ALM Accelerator is no longer actively maintained or supported. Therefore, this issue is being closed accordingly.
